content/en/docs/concepts/containers/images.md

```diff
@@ -33,17 +33,35 @@ Container images are usually given a name such as `pause`, `example/mycontainer`
 Images can also include a registry hostname; for example: `fictional.registry.example/imagename`,
 and possibly a port number as well; for example: `fictional.registry.example:10443/imagename`.
 
-If you don't specify a registry hostname, Kubernetes assumes that you mean the Docker public registry.
+If you don't specify a registry hostname, Kubernetes assumes that you mean the [Docker public registry](https://hub.docker.com/).
+You can change this behaviour by setting default image registry in [container runtime](/docs/setup/production-environment/container-runtimes/) configuration.
 
-After the image name part you can add a _tag_ (in the same way you would when using with commands
-like `docker` or `podman`).  Tags let you identify different versions of the same series of images.
+After the image name part you can add a _tag_ or _digest_ (in the same way you would when using with commands
+like `docker` or `podman`). Tags let you identify different versions of the same series of images.
+Digests are a unique identifier for a specific version of an image. Digests are hashes of the image's content,
+and are immutable. Tags can be moved to point to different images, but digests are fixed.
 
 Image tags consist of lowercase and uppercase letters, digits, underscores (`_`),
-periods (`.`), and dashes (`-`).  
-There are additional rules about where you can place the separator
-characters (`_`, `-`, and `.`) inside an image tag.  
+periods (`.`), and dashes (`-`). It can be up to 128 characters long. And must follow the
+next regex pattern: `[a-zA-Z0-9_][a-zA-Z0-9._-]{0,127}`
+You can read more about and find validation regex in the
+[OCI Distribution Specification](https://github.com/opencontainers/distribution-spec/blob/v1.1.0/spec.md#workflow-categories).
 If you don't specify a tag, Kubernetes assumes you mean the tag `latest`.
 
+Image digests consists of a hash algorithm (such as `sha256`) and a hash value. For example:
+`sha256:1ff6c18fbef2045af6b9c16bf034cc421a29027b800e4f9b68ae9b1cb3e9ae07`
+You can find more information about digests format in the 
+[OCI Image Specification](https://github.com/opencontainers/image-spec/blob/v1.1.0/descriptor.md#digests).
+
+Some image name examples that Kubernetes can use are:
```

Comment:
I guess it's great for understanding to have list of some examples with explanations

Reply:
I think busybox and busybox:1.32.0 share pattern of defining image names, so It's good to only mention one with ⇾ `Kubernetes will use Docker public registry and latest tag.` otherwise it also doesn't seem to have any issues.

Reply:
I guess that more examples are always better than less. So looks like it's a good place to mention about latest tag